vercel.json

```diff
@@ -93,7 +95,9 @@
 
     { "source": "/en/guides/debugging/", "destination": "/en/guides/troubleshooting/" },
 
-    { "source": "/en/quick-start/", "destination": "/en/installation/" },
+    { "source": "/en/quick-start/", "destination": "/en/install-and-setup/" },
+
+    { "source": "/en/installation/", "destination": "/en/install-and-setup/" },
```

Comment:
Why was this added? I don’t see a current `/installation/` link in use

Reply:
This was preexisting, and I thought it better to update to be safe? I'm not sure anyone might still be linking to `quick-start` at this point, but I noticed it pointing to something so thought I should update the link in the chain.